### PR TITLE
Fixing vtorc naming convention - Functions and Variables

### LINKED ISSUE #7110: Task: Cleanup references to master and slave in vt/orchestrator
There are several references in the code under `vt/orchestrator`. Some of them can probably be deleted outright because we don't have to worry about backwards-compatibility yet.

For example:
```
SlaveLagQuery                              string   // Synonym to ReplicationLagQuery
```
Variables, functions and parameters should be renamed.
Every occurrence should be evaluated to determine whether it can be removed. There might be exceptions like the exact text of MySQL error messages and low-level MySQL commands.

### PR DESCRIPTION
**Pull request by GuptaManan100**
## Description
Cleanup references to master and slave in `vt/orchestrator` in functions and variables 

## Related Issue(s)
- Fixes #7110

## Checklist
- [ ] Should this PR be backported?
- [x] Tests were added or are not required
- [x] Documentation was added or is not required

### CODE REVIEW

File: go/vt/orchestrator/inst/instance.go [hunk 1]
```diff
@@ -48,10 +48,10 @@ type Instance struct {
 	LogSlaveUpdatesEnabled       bool // for API backwards compatibility. Equals `LogReplicationUpdatesEnabled`
 	LogReplicationUpdatesEnabled bool
 	SelfBinlogCoordinates        BinlogCoordinates
-	MasterKey                    InstanceKey
-	MasterUUID                   string
+	PrimaryKey                   InstanceKey
```
Comment (shlomi-noach) [inline go/vt/orchestrator/inst/instance.go:51]: It's a minor thing, but it's confusing: `PrimaryKey` can be confused with `PRIMARY KEY`. What do you think?

**Review by shlomi-noach — APPROVED** [review]
This looks safe

File: go/vt/orchestrator/inst/instance.go (hunk 1 above)
Comment (GuptaManan100) [reply go/vt/orchestrator/inst/instance.go:51]: Should I make it `PrimaryInstanceKey`, I think that would clear it up

Comment (shlomi-noach) [reply go/vt/orchestrator/inst/instance.go:51]: yeah, maybe? Or `SourceKey`? I'm really not sure.

Comment (GuptaManan100) [reply go/vt/orchestrator/inst/instance.go:51]: We can make it `SourceKey` too. That also makes sense. Then we should also rename to `SourceUUID` for consistency. WDYT?

Comment (shlomi-noach) [reply go/vt/orchestrator/inst/instance.go:51]: let's ask @deepthi

Comment (deepthi) [reply go/vt/orchestrator/inst/instance.go:51]: Either is fine. In general using Source for replication related params makes sense.

Comment (GuptaManan100) [reply go/vt/orchestrator/inst/instance.go:51]: Alright i'll do that

Comment (GuptaManan100) [reply go/vt/orchestrator/inst/instance.go]: Done

**Pull request merged**